### PR TITLE
ide-usage.md: eclipse setup: correct menu path for project-specific settings #7136

### LINKED ISSUE #7136: ide-usage.md: eclipse setup: correct menu path for project-specific settings
In the ide-usage.md in [Step 4](https://github.com/TEAMMATES/teammates/blob/master/docs/ide-usage.md#eclipse-ide) it says 

Configure the following project-specific settings (all can be found in `Project → Properties → ...`):

```
HTML: Web → HTML Files → Editor → Indent using spaces.
CSS: Web → CSS Files → Editor → Indent using spaces.
XML: XML → XML Files → Editor → Indent using spaces.
```

but in both windows and Linux these option is there in  `Window`→ `Preference`

screenshot from 2017-04-16 03-38-01

### PR DESCRIPTION
**Pull request by ihsraham**
Fixes #7136  

Corrected 'Windows' to 'Windows and Linux' at line 8.
Added 'Eclipse -> Preferences ->' at line 67, 68 and 69.

### CODE REVIEW

**Comment by leeyimin** [comment]
Hi @ihsraham, thanks for contributing to TEAMMATES! Do take a look at our [orientation guide](https://github.com/TEAMMATES/teammates/blob/master/docs/orientation-guide.md) and [development process document](https://github.com/TEAMMATES/teammates/blob/master/docs/process.md) if you have not 😄 Do remember to use the issue title with the issue number as the PR title for your next PR!

File: docs/ide-usage.md [hunk 1]
```diff
@@ -64,9 +64,9 @@ The following plugins are needed:
      Configure for all the languages used in TEAMMATES:
      * Java: `Java → Code Style → Formatter → Edit → Tab policy → Spaces only`.
      * JavaScript: `JavaScript → Code Style → Formatter → Edit → Tab policy → Spaces only`.
-     * HTML: `Web → HTML Files → Editor → Indent using spaces`.
-     * CSS: `Web → CSS Files → Editor → Indent using spaces`.
-     * XML: `XML → XML Files → Editor → Indent using spaces`.
+     * HTML: `Eclipse → Preferences → Web → HTML Files → Editor → Indent using spaces`.
+     * CSS: `Eclipse → Preferences → Web → CSS Files → Editor → Indent using spaces`.
+     * XML: `Eclipse → Preferences → XML → XML Files → Editor → Indent using spaces`.
```
Comment (leeyimin) [inline docs/ide-usage.md]: You may want to edit line 46, to mention the exception about html, css, xml indentation. For this section, you may want to add a line above these 3 lines, mentioning the relevant place to change the settings are under `Eclipse -> Preferences`, and remove the `Eclipse -> Preferences` in these 3 lines instead.

Comment (ihsraham) [inline docs/ide-usage.md]: Thanks for your suggestions. I'll work on that right away :) .

Comment (ihsraham) [inline docs/ide-usage.md]: Would it be fine if I mentioned an exception for those three lines at line 46 ?

> 1. Configure the following project-specific settings (all can be found in `Project → Properties → ...` except for HTML, CSS and XML settings which can be found at 'Eclipse -> Preferences' ):

Comment (leeyimin) [inline docs/ide-usage.md]: I think it would be better to add an additional statement about it right before this part, since the reader would probably skim through that line or forgot about where they were supposed to go by the time they reached this section.

**Comment by leeyimin** [comment]
@ihsraham Any update on this?

**Comment by ihsraham** [comment]
@leeyimin I am extremely sorry for the delay as my laptop stopped working and I wasn't able to provide an update.
Would this be fine at line 67 before those three lines?

> (You can find Web and XML options in `Eclipse → Preferences`)
>      * HTML: `Web → HTML Files → Editor → Indent using spaces`.
>      * CSS: `Web → CSS Files → Editor → Indent using spaces`.
>      * XML: `XML → XML Files → Editor → Indent using spaces`.

**Comment by leeyimin** [comment]
How about
> You can find the Web and XML settings in `Eclipse → Preferences`. 
I would think that the brackets are not necessary.

File: docs/ide-usage.md [hunk 2]
```diff
@@ -43,7 +43,7 @@ The following plugins are needed:
    * Set the `root directory` to the location where the repo is cloned.
    * Click `Finish`.
 
-1. Configure the following project-specific settings (all can be found in `Project → Properties → ...`):
+1. Configure the following project-specific settings (all can be found in `Project → Properties → ...` except for HTML, CSS and XML settings which can be found at 'Eclipse -> Preferences'):
```
Comment (leeyimin) [inline docs/ide-usage.md]: Add a comma before 'except'.

Comment (leeyimin) [inline docs/ide-usage.md]: `Eclipse -> Preferences -> ...` instead of 'Eclipse -> Preferences'

File: docs/ide-usage.md [hunk 3]
```diff
@@ -64,9 +64,11 @@ The following plugins are needed:
      Configure for all the languages used in TEAMMATES:
      * Java: `Java → Code Style → Formatter → Edit → Tab policy → Spaces only`.
      * JavaScript: `JavaScript → Code Style → Formatter → Edit → Tab policy → Spaces only`.
-     * HTML: `Eclipse → Preferences → Web → HTML Files → Editor → Indent using spaces`.
-     * CSS: `Eclipse → Preferences → Web → CSS Files → Editor → Indent using spaces`.
-     * XML: `Eclipse → Preferences → XML → XML Files → Editor → Indent using spaces`.
+     
+     You can find Web and XML options in `Eclipse → Preferences`
```
Comment (leeyimin) [inline docs/ide-usage.md]: Add 'the' before `Web and XML...'.

Comment (leeyimin) [inline docs/ide-usage.md]: Add a full stop.

File: docs/ide-usage.md [hunk 4]
```diff
@@ -43,7 +43,7 @@ The following plugins are needed:
    * Set the `root directory` to the location where the repo is cloned.
    * Click `Finish`.
 
-1. Configure the following project-specific settings (all can be found in `Project → Properties → ...`):
+1. Configure the following project-specific settings (all can be found in `Project → Properties → ...`, except for HTML, CSS and XML settings which can be found at `Eclipse -> Preferences`):
```
Comment (leeyimin) [inline docs/ide-usage.md]: Use `Eclipse -> Preferences -> ...` to standardise such references.

File: docs/ide-usage.md [hunk 5]
```diff
@@ -64,6 +64,8 @@ The following plugins are needed:
      Configure for all the languages used in TEAMMATES:
      * Java: `Java → Code Style → Formatter → Edit → Tab policy → Spaces only`.
      * JavaScript: `JavaScript → Code Style → Formatter → Edit → Tab policy → Spaces only`.
+     
+     You can find the Web and XML options in `Eclipse → Preferences`.
```
Comment (leeyimin) [inline docs/ide-usage.md]: Here as well.

File: docs/ide-usage.md [hunk 6]
```diff
@@ -43,7 +43,7 @@ The following plugins are needed:
    * Set the `root directory` to the location where the repo is cloned.
    * Click `Finish`.
 
-1. Configure the following project-specific settings (all can be found in `Project → Properties → ...`):
+1. Configure the following project-specific settings (all can be found in `Project → Properties → ...`, except for the HTML, CSS and XML settings which can be found at `Eclipse -> Preferences -> ...`):
```
Comment (leeyimin) [inline docs/ide-usage.md]: Sorry for my mistake, I used the wrong symbols in my comments, do replace `->` with `→`.

File: docs/ide-usage.md [hunk 7]
```diff
@@ -64,6 +64,8 @@ The following plugins are needed:
      Configure for all the languages used in TEAMMATES:
      * Java: `Java → Code Style → Formatter → Edit → Tab policy → Spaces only`.
      * JavaScript: `JavaScript → Code Style → Formatter → Edit → Tab policy → Spaces only`.
+
```
Comment (whipermr5) [inline docs/ide-usage.md]: Hang on; there are trailing spaces here.

File: docs/ide-usage.md [hunk 8]
```diff
@@ -64,8 +64,7 @@ The following plugins are needed:
      Configure for all the languages used in TEAMMATES:
      * Java: `Java → Code Style → Formatter → Edit → Tab policy → Spaces only`.
      * JavaScript: `JavaScript → Code Style → Formatter → Edit → Tab policy → Spaces only`.
-     
-     You can find the Web and XML options in `Eclipse → Preferences → ...`.
+     <br>You can find the Web and XML options in `Eclipse → Preferences → ...`.
```
Comment (whipermr5) [inline docs/ide-usage.md]: Oops I realised the trailing spaces were to maintain the indentation; my bad 😅 Can you revert the change?

Comment (ihsraham) [inline docs/ide-usage.md]: No problem ;-)

File: docs/ide-usage.md [hunk 9]
```diff
@@ -64,6 +64,7 @@ The following plugins are needed:
      Configure for all the languages used in TEAMMATES:
      * Java: `Java → Code Style → Formatter → Edit → Tab policy → Spaces only`.
      * JavaScript: `JavaScript → Code Style → Formatter → Edit → Tab policy → Spaces only`.
+     <br>You can find the Web and XML options in `Eclipse → Preferences → ...`.
```
Comment (whipermr5) [inline docs/ide-usage.md]: I meant to the original version without a `<br>`,  but with a blank line containing trailing spaces 😅

Comment (ihsraham) [inline docs/ide-usage.md]: Yeah got it :smile:

**Comment by damithc** [comment]
Let me know when ready for my review.

**Comment by whipermr5** [comment]
@damithc Ready!

**Comment by ihsraham** [comment]
Was my first PR. Thanks for your reviews @leeyimin @whipermr5 , would keep them in mind henceforth :wink:

**Comment by thenaesh** [comment]
@ihsraham May we have your full name for inclusion into the contributors' list?

**Comment by ihsraham** [comment]
Sure. My full name is Maharshi Mishra.
